### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ycao56**
https://github.com/open-cluster-management/backlog/issues/11335

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,40 +3,114 @@
...
-## Bring your own policy
+The governance policy framework consists of following components:
+
+- [Governance dashboard](https://github.com/open-cluster-management/grc-ui) -- UI
+- Govenance policy framework -- A framework to distribute various supported policies to managed cluster and collect results back to hub. It consists of following components:
+    - [Policy propagator](https://github.com/open-cluster-management/governance-policy-propagator) 
+    - [Policy spec sync](https://github.com/open-cluster-management/governance-policy-spec-sync)
+    - [Policy status sync](https://github.com/open-cluster-management/governance-policy-status-sync)
+    - [Policy template sync](https://github.com/open-cluster-management/governance-policy-template-sync)
+- Policy controllers -- Policy engines running on managed cluster to evaluate policy rules distributed by the policy framework and generate results.
```
Comment (dockerymick) [inline README.md:31]: I suggest to update "Policy engines running on managed cluster to evaluate policy rules distributed by the policy framework and generate results."

TO 

Policy engines that run on managed clusters to evaluate policy rules distributed by the policy framework and generate results.

File: README.md [hunk 2]
```diff
@@ -3,40 +3,114 @@
...
+    - [Configuration policy controller](https://github.com/open-cluster-management/config-policy-controller)
+    - [Certificate policy controller](https://github.com/open-cluster-management/cert-policy-controller)
+    - [IAM policy controller](https://github.com/open-cluster-management/iam-policy-controller)
+    - 3rd party
```
Comment (dockerymick) [inline README.md:35]: I recommend changing 3rd party to Third-party

File: README.md [hunk 3]
```diff
@@ -3,40 +3,114 @@
...
+      - [Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
+      - [Kyverno](https://github.com/kyverno/kyverno/)
+      - [Bring your own](#bring-your-own-policy-controller)
+
+## The Policy CRDs
+
+The `Policy` is the Custom Resource Definition (CRD), created for policy framework controllers to monitor. It acts as a vechical to deliver policies to managed cluster and collect results back to hub.
```
Comment (dockerymick) [inline README.md:42]: Update vechical to **vehicle**

Comment (dockerymick) [reply README.md:42]: ...collect results to send to the hub cluster

File: README.md [hunk 4]
```diff
@@ -3,40 +3,114 @@
...
+This is an example spec of a `Policy` object:
+```yaml
```
Comment (dockerymick) [inline README.md:44]: I recommend that you update "This is an example spec of a `Policy` object:" 

TO 

"View the following example specification of a `Policy` object"

File: README.md [hunk 5]
```diff
@@ -3,40 +3,114 @@
...
+                  namespace: default
+                spec:
+                  containers:
+                  - image: nginx:1.7.9
+                    name: nginx
+                    ports:
+                    - containerPort: 80
+```
+
+The `PlacementBinding` CRD is used to bind the `Policy` with `PlacementRule`. Only bound `Policy` will be distributed by policy framework to the managed cluster
```
Comment (dockerymick) [inline README.md:76]: Change: The `PlacementBinding` CRD is used to bind the `Policy` with `PlacementRule`.

TO 

The `PlacementBinding` CRD is used to bind the `Policy` with a `PlacementRule`.

Comment (dockerymick) [reply README.md:76]: Only a bound `Policy` is distributed to a managed cluster by the policy framework.

File: README.md [hunk 6]
```diff
@@ -3,40 +3,114 @@
...
+This is an example spec of a `PlacementBinding` object:
```
Comment (dockerymick) [inline README.md:77]: View the following example specification of a `PlacementBinding` object:

File: README.md [hunk 7]
```diff
@@ -3,40 +3,114 @@
...
+  kind: PlacementRule
+  apiGroup: apps.open-cluster-management.io
+subjects:
+- name: policy-pod
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+```
+
+The `PlacementRule` CRD is used to determine the target clusters to distribute policies to.
+This is an example spec of a `PlacementRule` object:
```
Comment (dockerymick) [inline README.md:94]: View the following example specification of a `PlacementRule` object:

**Review by dockerymick — CHANGES_REQUESTED** [review]

**Comment by ycao56** [comment]
all done. merging it

**Pull request merged**